cookbook-react/recipes/inputs/barcelona/index.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import cn from 'classnames';
+
+import styles from './styles.module.scss';
+import { ReactComponent as ValidIcon } from './assets/ic_valid.svg';
+import { ReactComponent as InvalidIcon } from './assets/ic_invalid.svg';
+
+interface Props {
+  label?: string;
+  disabled?: boolean;
+  error?: string;
+  placeholder?: string;
+  confirmable?: boolean;
+  name: string;
+  onChange?: (event: React.FormEvent<HTMLInputElement>) => void;
+}
+
+const INPUT_CONTAINER_ID = "barcelona"
+const INPUT_ID = `${INPUT_CONTAINER_ID}-input`
+
+function Input({ name, label, disabled, error, onChange, confirmable = true, placeholder }: Props) {
+  return (
+    <div id={INPUT_CONTAINER_ID}>
```

Comment:
Why do you need this `id`? and the `div`?

Reply:
I just copied the other input. I thought this was used to render it in the cookbook or something but I asked Tincho and it's not used for anything so I'll remove the id and the div

Reply:
Yeah, we should probably check the other input then.